tests/regression/lp-1644439/kernel-new-enough.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# Check if the kernel is at least 4.4.0-67
+if [ "$(uname -r | perl -ne 'print $1 if /^([0-9]+)\./')" -lt 4 ]; then
+	exit 1
+fi
+if [ "$(uname -r | perl -ne 'print $1 if /^[0-9]+\.([0-9]+)\./')" -lt 4 ]; then
+	exit 1
+fi
+if [ "$(uname -r | perl -ne 'print $1 if /^[0-9]+\.[0-9]+\.[0-9]+-([0-9]+)/')" -lt 67 ]; then
+	exit 1
+fi
```

Comment:
My "ew" is because of how this manages to be both inefficient *and* hard to read :-)
It's so hard to read I *think* it's wrong, but I'm not certain: is your intention that 4.4.0-67 is OK but 4.4.1-30 is not?